etna/search/tests/test_views.py

```diff
@@ -784,20 +794,50 @@ def test_current_bucket(self):
                     result_count=8,
                     is_current=False,
                     results=None,
+                    aggregations=(
+                        Aggregation.COLLECTION,
+                        Aggregation.LEVEL,
+                        Aggregation.TOPIC,
+                        Aggregation.CLOSURE,
+                        Aggregation.HELD_BY,
+                        Aggregation.CATALOGUE_SOURCE,
+                        Aggregation.GROUP + ":30",
+                        Aggregation.TYPE,
+                    ),
```

Comment:
If this is the same as `DEFAULT_BUCKETS`, there's no need to specify this here any longer (same for all tests)